evm_arithmetization/src/cpu/columns/general.rs

```diff
@@ -142,6 +155,14 @@ pub(crate) struct CpuShiftView<T: Copy> {
     pub(crate) high_limb_sum_inv: T,
 }
 
+/// View of the first `CpuGeneralColumns` storing a flag for context pruning.
+#[derive(Copy, Clone)]
+pub(crate) struct CpuContextPruningView<T: Copy> {
+    /// If `context_op` is set, is 1 if the operation is `SET_CONTEXT` and
```

Comment:
Where is `context_op` coming from? If the reader doesn't know, then this phrase is difficult to parse.

Reply:
```suggestion
    /// If `context_op` is set, then this is 1 if the operation is `SET_CONTEXT` and
```